tests/unittests/triton/test_pipeline_client.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2021 Zilliz. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import json
+import numpy as np
+
+from towhee.utils.np_format import NumpyArrayEncoder
+from towhee.serve.triton.pipeline_client import Client
+
+
+class MockHttpClient:
+    """
+    mock http client
+    """
+    def __init__(self, url):
+        self._url = url
+
+    def infer(self, model_name, inputs):
+        res = inputs[0].shape()
+        return MockRes(model_name, res)
+
+    def close(self):
+        pass
+
+
+class MockRes:
+    """
+    mock response class for triton client
+    """
+    def __init__(self, model_name: str, outputs):
+        self._name = model_name
+        self._outputs = outputs
+
+    def as_numpy(self, name):
+        data = json.dumps((self._name, name, self._outputs), cls=NumpyArrayEncoder)
+        return np.array([data], dtype=np.object_)
+
+
+class TestTritonClient(unittest.TestCase):
```

Comment:
refer to this test: `towhee/tests/unittests/triton/bls/test_pipeline_model.py`, write a better test.